src/plugins/gitvcschanges/gitvcschangesplugin.h

```diff
@@ -0,0 +1,59 @@
+/****************************************************************************
+**
+** Copyright (C) 2016 The Qt Company Ltd.
+** Contact: https://www.qt.io/licensing/
```

Comment:
I dont think that new files from you should have this copyright notice from Qt Company.